src/bankid.ts

```diff
@@ -195,10 +195,12 @@ export class BankIdClient {
     this.options = {
       production: false,
       refreshInterval: 2000,
-      // defaults for test environment
-      passphrase: "qwerty123",
-      pfx: path.resolve(__dirname, "../cert/", "FPTestcert3_20200618.p12"),
       ...options,
+      // defaults for test environment
+      passphrase: options?.passphrase ?? "qwerty123",
+      pfx:
+        options?.passphrase ??
+        path.resolve(__dirname, "../cert/", "FPTestcert3_20200618.p12"),
```

Comment:
I would rather not provide these values for production mode.

Can you change the implementation to how it is like for `this.options.ca` in L214?

Reply:
Right, functionality-wise it would be the same (missing pfx or passphrase when production is set would throw) because that condition is looking at `options` rather than `this.options` but i agree it will be more explicit this way.

I added an else to the `if (production)` condition where i check for undefined passpharse/pfx and assign them to test defaults :smile: :+1: